package.json

```diff
@@ -10,14 +10,86 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:neo4j-cypher-runner.run-query"
+    "onCommand:neo4j-cypher-runner.run-query",
+    "onCommand:neo4j-cypher-runner.select-database"
```

Comment:
nit: maybe `target` is less ambiguous than `database`? Database is quite overloaded since Neo4j supports multi-tenancy (applies to other uses of `database` below as well)

Reply:
Fixed.